lib/coordinate.rb

```diff
@@ -5,4 +5,12 @@ def initialize(x = 0, y = 0)
         @x = x
         @y = y
     end
+
+    def inspect
```

Comment:
This was to improve the failure message. 
The failure message was showing a Coordinate object with no details about the data. This is the way to tell ruby how to show the coordinate object when there is a failure